src/java/main/org/apache/zookeeper/ClientCnxnSocket.java

```diff
@@ -63,6 +65,12 @@
     protected long now;
     protected ClientCnxn.SendThread sendThread;
     protected LinkedBlockingDeque<Packet> outgoingQueue;
+    protected ZKClientConfig clientConfig;
+    /**
```

Comment:
We have used mostly comments on top of the line, but I don't mind if you put it at the end of the line. I also agree that the use of a block comment here isn't entirely appropriate. It should be either `/* 4MB */`or `// 4MB`.